sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/SentryTaskDecorator.java

```diff
@@ -14,18 +15,13 @@
  */
 public final class SentryTaskDecorator implements TaskDecorator {
   @Override
-  @SuppressWarnings("deprecation")
+  // TODO should there also be a SentryIsolatedTaskDecorator or similar that uses forkedScopes()?
```

Comment:
Should we create a `SentryIsolatedTaskDecorator`?

Reply:
Not even sure what the better default here is, isolated or not?

Reply:
I think it should be the isolation scope, as otherwise (when just forking the current scope) any scope changes made in the user code with the static API (`Sentry.setTag()`) would leak outside the task boundaries

Reply:
Yeah, the thing here is, that methods annotated with `@Async` also use this iirc and I'm not so sure about isolating for that use case.